### ISSUE TITLE
pprint for hyperparameter printout

### ISSUE THREAD

**Issue by Anselmoo** [keras-team/autokeras#1690]
I am wondering if [pprint](https://docs.python.org/3/library/pprint.html#module-pprint) can be used for the metric printout?

> Sometimes the results of the solver are too harsh cropped? 

<img width="502" alt="image" src="https://user-images.githubusercontent.com/13209783/154848699-3a1b562e-86b9-4792-beff-f2debabdf266.png">

`pprint` could provide a screen optimized printout and is also builtin-function.

https://github.com/keras-team/autokeras/blob/85bf1e28e39b4e12da795bce88b9a93bb92ef213/autokeras/tuners/task_specific.py#L17-L33
